lib/bitfields.rb

```diff
@@ -141,10 +141,12 @@ def bitfield_sql_by_column(column, bit_values, options={})
         "(#{table_name}.#{column} & #{on+off}) = #{on}"
       when :bit_operator_or
         result = []
-        bit_values.each do |bit_name, value|
-          bit = bitfields[column][bit_name]
-          eql = value ? bit : 0
-          result << "(#{table_name}.#{column} & #{bit}) = #{eql}"
+        on, off = bit_values_to_on_off(column, bit_values)
+        if on != 0
+          result << "(#{table_name}.#{column} & #{on}) <> 0"
```

Comment:
- extract `#{table_name}.#{column}` to a variable ?
 - `result << foo if on != 0` <- looks better ?

Reply:
1. I don't think it will be some speed or readability improvement if we extract variable. And if extract it, then we can do that for whole method.

2. If operations updated